backend/index.js

```diff
@@ -18,4 +18,18 @@ app.get('/getAnimalDescription', function (req, res) {
     })
 })
 
+app.get('/getMatches', function(req, res){
+    axios.get('https://codubee-projects-api.herokuapp.com/animal/getMatches')
```

Comment:
/getMatches doesn't getMatches based on user id given by req in fucntion(req,res). ie. https://codubee-projects-api.herokuapp.com/animal/getMatches?id=1 so when I test it localhost:8080/getMatches always returns no matches.

Reply:
I agree with Rosey. I think you have to add a query parameter to your route here. You can take a look at [this](https://github.com/Codubee/Node-Js-API/blob/39babb7639c29b9c0f5fe50ae72f1e7b6e737697/index.js#L37) example as a guide